labels/mapping.go

```diff
@@ -1,7 +1,33 @@
 package main
 
-var mapping = map[string][]Label{
-	"mattermost-plugin-giphy": MergeLabels(core, plugin, helpWanted),
+var pluginLabels = MergeLabels(core, plugin, helpWanted, docs)
+
+var pluginRepos = []string{
+	"mattermost-oembed-plugin",
+	"mattermost-plugin-antivirus",
+	"mattermost-plugin-autolink",
+	"mattermost-plugin-aws-SNS",
+	"mattermost-plugin-custom-attributes",
+	"mattermost-plugin-giphy",
+	"mattermost-plugin-giphy",
+	"mattermost-plugin-github",
+	"mattermost-plugin-gitlab",
+	"mattermost-plugin-jenkins",
+	"mattermost-plugin-jira",
+	"mattermost-plugin-mailermost",
+	"mattermost-plugin-suggestions",
+	"mattermost-plugin-webex",
+	"mattermost-plugin-welcomebot",
+	// "mattermost-plugin-workflow",
+	"mattermost-plugin-zoom",
+}
```

Comment:
What was your guideline for picking these plugins? Why was e.g. mattermost-plugin-memes leaved out?

Reply:
I just added the repos that I track/fully know about. Generally correlates with what I have been considering "production". I have no objection to extending the list, but also maybe 1/5 we should make the repo name an (optional) command flag? (defaults to the list)

Reply:
I like the idea :+1:. Let's go with this list (plus mattermost-plugin-workflow?) for now and iterate on it.